tests/Resources/ti.ui.navigationwindow.test.js

```diff
@@ -9,45 +9,184 @@
 'use strict';
 const should = require('./utilities/assertions');
 
-describe.windowsMissing('Titanium.UI.NavigationWindow', function () {
+describe('Titanium.UI.NavigationWindow', function () {
 	this.timeout(10000);
 
 	let nav;
-	afterEach(function () {
-		if (nav) {
-			nav.close();
+	afterEach(done => {
+		if (nav && !nav.closed) {
+			nav.close().then(() => done()).catch(_e => done());
```

Comment:
<!--
  0 failure: 
  2 warning:  tests/Resources/t..., tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.navigationwindow.test.js;
  Line: 18;
-->

- :warning: tests/Resources/ti.ui.navigationwindow.test.js line 18 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)
- :warning: tests/Resources/ti.ui.navigationwindow.test.js line 18 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)